dev-python/transitions: bump to version 0.7.1

ConiKost opened the pull request:
aPackage-Manager: Portage-2.3.76, Repoman-2.3.17
Signed-off-by: Conrad Kostecki <conrad@kostecki.com>

juippis commented:
```
>>> Source compiled.
>>> Test phase: dev-python/transitions-0.7.1
 * python2_7: running distutils-r1_run_phase python_test
python2.7 setup.py test
running test
Searching for graphviz
Reading https://pypi.org/simple/graphviz/
Download error on https://pypi.org/simple/graphviz/: [Errno -3] Temporary failure in name resolution -- Some packages may not be found!
Couldn't find index page for 'graphviz' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading https://pypi.org/simple/
Download error on https://pypi.org/simple/: [Errno -3] Temporary failure in name resolution -- Some packages may not be found!
No local packages or working download links found for graphviz
error: Could not find suitable distribution for Requirement.parse('graphviz')
 * ERROR: dev-python/transitions-0.7.1::testworld failed (test phase):
 *   (no error message)
```

Tests succeeded when I emerge dev-python/graphviz, HOWEVER it doesnt have python3.7 support yet so that died.

ConiKost commented:
Added missing ```dev-python/graphviz``` into ```DEPEND``` for ```test?```, should fix that issue. Also updated ```dev-python/graphviz```to latest version and it seems to run with python3.7 fine and tests pass here for both packages just fine.
